tensorrt_llm/serve/openai_client.py

```
@@ -475,6 +483,33 @@ async def check_ready(self) -> Tuple[List[str], List[str]]:
+            else f"http://{server}/v1/data_transceiver_state"
+        )
+        state = None
+        try:
+            async with self._session.get(url) as response:
+                if response.status == 200:
+                    state = (await response.json()).get("data_transceiver_state")
+                else:
+                    logger.warning(
+                        f"{server} returned {response.status} for data_transceiver_state"
+                    )
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)